webapp/packages/plugin-data-viewer/src/ValuePanelPresentation/TextValue/TextValuePresentation.tsx

```diff
@@ -115,7 +115,13 @@ export const TextValuePresentation: TabContainerPanelComponent<IDataValuePanelPr
     const autoLineWrapping = getDefaultLineWrapping(contentType);
     const lineWrapping = state.lineWrapping ?? autoLineWrapping;
 
-    const { textValue, isTruncated, isTextColumn, pasteFullText } = useTextValue({
+    const {
+      textValue,
+      isTruncated,
+      isTextColumn,
+      pasteFullText,
+      limitString: limit,
+    } = useTextValue({
```

Comment:
now this looks really bad lets replace it with `textValueData` const and access properties from this object